src/redux/appReducer.js

```diff
@@ -0,0 +1,26 @@
+import * as buttonActions from '../pressButtonLogic';
+import * as types from './actionTypes';
+
+export default function appReducer(
+    state = {
+        result: ['0'],
```

Comment:
When this value changes the display should re-render. No special action required for this to happen.